Formula/rpki-client.rb

```diff
@@ -0,0 +1,49 @@
+class RpkiClient < Formula
+  desc "OpenBSD portable rpki-client"
+  homepage "https://github.com/rpki-client/rpki-client-portable"
+  url "https://github.com/rpki-client/rpki-client-portable/archive/refs/tags/7.4.tar.gz"
+  sha256 "cacb56107b29baef67c0117bc53d726c95684ad27a6affe2a1527470c67f5e86"
+  license "ISC"
+
+  depends_on arch: :intel
+  depends_on "autoconf"
+  depends_on "automake"
+  depends_on "libressl"
```

Comment:
Can we use OpenSSL?

Reply:
No, I believe there is a specific dependency on libressl which the rpki-client Authors told me required libtls from this package, and some associated fixes which are in libressl and not in openssl.